app/model/classifications.js

```diff
@@ -324,6 +330,31 @@ var Classifications = {
             JOIN models m on m.model_id = mc.model_id
             WHERE m.project_id = ${dbpool.escape(projectId)} AND m.deleted = 0`).get(0).get('count');
     },
+
+    JOB_SCHEMA : joi.object().keys({
+        ENV_JOB_ID: joi.string()
+    }),
+
+    createClassificationJob: function(data, callback){
+        console.log('data', data)
+        const payload = JSON.stringify(
+            {
+                ENV_JOB_ID: `${data.jobId}`
+            }
+        )
+        return q.ninvoke(joi, 'validate', payload, Classifications.JOB_SCHEMA)
+            .then(async () => {
+                data.kubernetesJobName = `arbimon-classification-job-${new Date().getTime()}`;
+                const jobParam = jsonTemplates.getClassificationJobTemplate('arbimon-classification-job', 'job', {
+                    kubernetesJobName: data.kubernetesJobName,
+                    imagePath: k8sConfig.soundscapeImagePath,
```

Comment:
this is the problem, you have to use `arbimon-rfm`

Reply:
fixed